src/Makefile

```diff
@@ -110,7 +112,7 @@ memento_test_LDFLAGS := ${COMMON_LDFLAGS} -lcurl
 
 include ../build-infra/cpp.mk
 
-ROOT := $(abspath $(shell pwd)/../)
+ROOT := ..
```

Comment:
I'm not changing this for `cpp.mk` here, I'm changing it for `alarm-utils.mk` a couple of lines below (but not show by github by default).

Yes, the build is fine with this.